app/assets/javascripts/controllers/alerts/alerts_overview_controller.js

```diff
@@ -0,0 +1,262 @@
+/* global miqHttpInject */
+
+angular.module('alertsCenter').controller('alertsOverviewController',
+  ['$scope', '$window', 'alertsCenterService', '$interval', '$timeout',
+  function($scope, $window, alertsCenterService, $interval, $timeout) {
+    var vm = this;
+    vm.alertData = [];
+    vm.loadingDone = false;
+
+    function setupInitialValues() {
+      angular.element(document.querySelector('#center_div')).addClass("miq-body");
```

Comment:
Could you update this to `document.getElementById("center_div").className += " miq-body";` please? Or change the other two to this, I don't care as long as they're the same.